README.rst

```diff
@@ -27,6 +27,26 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
 This is easily achieved by downloading
 `the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
+Installing from PyPI
+====================
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-circuitpython-rtttl/>`_. To install for current user:
+
+.. code-block:: shell
+    pip3 install adafruit-circuitpython-rtttl
```

Comment:
Spacing incorrect on this and all following code blocks in this section. Please correct it.